src/internal/styles/utils/styles-reset.scss

```diff
@@ -14,12 +14,9 @@
   direction: inherit;
   empty-cells: show;
   font-family: serif;
-  font-size: medium;
```

Comment:
I realized that these 3 rules were unnecessary because they are overriden by the [`typography.default-text-style`](https://github.com/cloudscape-design/components/blob/main/src/internal/styles/typography/mixins.scss#L16-L19) mixin which is included further down